modules/network/kademlia/kademlia.js

```diff
@@ -59,43 +59,40 @@ class Kademlia {
             this.config.appDataPath,
             this.config.identity_filepath,
         );
+
         if (fs.existsSync(identityFilePath)) {
+            this.log.info('Using old identity...');
```

Comment:
Just a cosmetic change, but `existing` is a more appropriate term, let's use that